tools/shared/src/main/scala/org/scalajs/core/tools/linker/LinkedClass.scala

```diff
@@ -161,10 +161,11 @@ object LinkedClass {
 
       // Normal methods
       case m: MethodDef if m.name.isInstanceOf[Ident] =>
-        if (m.body == EmptyTree)
-          abstractMethods += linkedMethod(m)
-        else
+        if (m.body.isDefined) {
           memberMethods += linkedMethod(m)
+        } else {
+          abstractMethods += linkedMethod(m)
+        }
```

Comment:
Hmmm... I thought ifs with side-effects must have braces. Can't find this rule in the style guide though...